### DISCUSSION TITLE
UVMF using PyUVM

### DISCUSSION THREAD

**Discussion by hossamfadeel** [pyuvm/pyuvm#222] [General]
Hello Ray Salemi,

Is there any support for the UVMF code generator to be used alongside with the pyuvm and CocoTB?

Regards,

**Comment by raysalemi** [comment]
It sounds like you have defined an open-source project for yourself!

Truth be told, Python-based UVM doesn't need UVMF.  It is easy enough to write the code directly because there are no parameters.
